src/Npgsql/Internal/Converters/Temporal/DateConverters.cs

```diff
@@ -4,88 +4,88 @@
 // ReSharper disable once CheckNamespace
 namespace Npgsql.Internal.Converters;
 
-sealed class DateTimeDateConverter(bool dateTimeInfinityConversions) : PgBufferedConverter<DateTime>
+sealed class DateOnlyDateConverter(bool dateTimeInfinityConversions) : PgBufferedConverter<DateOnly>
```

Comment:
Why are we even move things around?

Reply:
Just to put the main/default converter first, is all...